README.md

```diff
@@ -2,6 +2,52 @@
 
 A Kubernetes Operator based on the Operator SDK for creating and managing Grafana instances.
 
+
+## Grafana Operator is utilised by the following:
```

Comment:
Something seems a bit off about the wording, looking for a better way to phrase this :thinking:

Reply:
You could maybe say something like "Companies that trust and use the Grafana operator"

Reply:
Yup, that sounds much better :+1: